app/pages/sections/order-items/additional-services/select/recipient/contextCreator.js

```diff
@@ -24,3 +25,16 @@ export const getContext = ({
   questions: recipients && generateQuestionsContext({ recipients }),
   backLinkHref: `${baseUrl}/organisation/${orderId}/additional-services/select/additional-service/price`,
 });
+
+
+export const getErrorContext = (params) => {
+  const updatedManifest = getContext({
+    orderId: params.orderId,
+    solutionName: params.itemName,
```

Comment:
should that still be solutionName? or is this what you meant earlier and that needs to change to itemName?